src/rez/bind/hello_world.py

```diff
@@ -43,7 +43,15 @@ def bind(path, version_range=None, opts=None, parser=None):
     def make_root(variant, root):
         binpath = make_dirs(root, "bin")
         filepath = os.path.join(binpath, "hello_world")
-        create_executable_script(filepath, hello_world_source)
+
+        # In order for this script to run on each platform we create the
```

Comment:
While this comment is correct, I don't think it belongs here. This caveat is already pointed out in a very similar comment with the function definition.